massa-bootstrap/src/client_binder.rs

```diff
@@ -84,11 +87,36 @@ impl BootstrapClientBinder {
             u32::from_be_bytes_min(&meg_len_bytes, self.max_bootstrap_message_size)?.0
         };
 
-        // read message, check signature and deserialize
+        // read message, check signature and, optionally, signature of the message we sent just before and deserialize
```

Comment:
This sentence doesn't make much sense to me, did you mean to say "and optionally check signature of the message sent just before then deserialize it"?

Reply:
Yes. My bad. Fixed.